acceptance/ui/features/steps/generic_steps.rb

```diff
@@ -115,7 +115,7 @@ def checkColumn(text, column, present)
     cell = getTableValue(text)
     hasEntry = false
     for i in 0..(list.size - 1)
-        hasEntry = true if list[i].text == cell
+        hasEntry = true if list[i].text == cell.to_s
```

Comment:
Convert to string, in case it's looking for the number of cores or some other int.